src/structs.jl

```diff
@@ -50,3 +50,173 @@ IPUMS.IPUMSSource("IPUMS USA", "usa", "microdata", "", true, "")
         home_url::String = ""
         IPUMSSource(proj_name::String, url_name::String, collection_type::String, code_for_api::String, api_support::Bool, home_url::String) = new(proj_name, url_name, collection_type, code_for_api, api_support, home_url)
 end
+
+
+"""
+```julia
+DDIVariable(
+    name::String = "", 
+    position_start::Int64 = 9999,
+    position_end::Int64 = 9999,
+    position_width::Int64 = 9999,
+    labl::String = "", 
+    desc::String = "", 
+    dcml::Int64 = 9999, 
+    vartype::String = "",
+    varinterval::String = ""
+)
+```
+
+A struct representing individual variable/column metadata from an IPUMS extract
+file. This stuct is used for parsing the IPUMS datafile, which may be of fixed 
+width format, hence the `position_` fields. The default value for missing 
+strings is a space `" "`, while the default value for missing integer values 
```

Comment:
You say space, but what is recorded actually shows an empty string. Was this a small typo?

Reply:
Yep, you are correct. I fixed this so it says "empty string" now.